plugins/remote-config/frontend/public/templates/parameters.html

```diff
@@ -12,14 +12,13 @@ <h2 class="cly-vue-remote-config__header"> {{i18n('remote-config.title')}}
     </template>
   </cly-header>
   <cly-main>
-    <json-editor></json-editor>
     <cly-datatable-n :rows="tableRows">
       <template v-slot:default="scope">
         <el-table-column type="expand" min-width="50" v-if="isDrillEnabled">
```

Comment:
if drill is disabled,
will this code block show the parameters default value?

Reply:
correct me if i am not wrong, due to this check, the table won't have an expandable column if drill is disabled.